tests/apitests/python/test_assign_sys_admin.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+
+import unittest
+
+from testutils import ADMIN_CLIENT
+from testutils import TEARDOWN
+from library.user import User
+from library.configurations import Configurations
+
+class TestProjects(unittest.TestCase):
+    @classmethod
+    def setUp(self):
+        conf = Configurations()
+        self.conf= conf
+
+        user = User()
+        self.user= user
+
+    @classmethod
+    def tearDown(self):
+        print "Case completed"
+
+    @unittest.skipIf(TEARDOWN == False, "Test data should be remain in the harbor.")
+    def test_ClearData(self):
+        #1. Delete user(UA);
+        self.user.delete_user(TestProjects.user_assign_sys_admin_id, **ADMIN_CLIENT)
+
+    def testAssignSysAdmin(self):
+        """
+        Test case:
+            Assign Sys Admin
```

Comment:
s/Sys Admin/sysadmin

Reply:
Hi clouderati, could you tell me more about this issue, I'm not quite clear on it.

Reply:
@clouderati  Thank you, I still have one issue, and need your input.